android/assets/jsons/Civ V - Gods & Kings/Beliefs.json

```diff
@@ -275,7 +275,7 @@
     {
         "name": "Messiah",
         "type": "Enhancer",
-        "uniques": ["[+25]% Spread Religion Strength <for [Great Prophet] units>", "[-25]% Faith cost of generating Great Prophet equivalents", "[Faith] cost for [Great Prophet] units [-25]%"]
```

Comment:
Why was this removed?

Reply:
As far as I was able to tell, its only appearance in the code was for AI automation, so it was already broken with no game rule effects, and I chose to deprecate and remove it for now rather than mixing a new implementation into this PR.

https://github.com/yairm210/Unciv/blob/6bc41a627c1118d8cb4736757b09515efe925ea6/core/src/com/unciv/logic/automation/ChooseBeliefsAutomation.kt#L180-L181

https://github.com/yairm210/Unciv/blob/3ce33fa1cfed1c01d6444a6bd1f39ce91979b7d3/core/src/com/unciv/models/ruleset/unique/UniqueType.kt#L847-L848

IMO Implementing it would best be a separate PR with more thorough testing and review than for the 1:1 conversion of existing logic I'm doing here.

Reply:
If I'm not mistaken, it's already in the UniqueType as `BuyUnitsDiscount`